core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -570,14 +570,7 @@ valueExpression
     ;
 
 primaryExpression
-    : NULL                                                                                #nullLiteral
-    | interval                                                                            #intervalLiteral
-    | identifier string                                                                   #typeConstructor
-    | DOUBLE PRECISION string                                                             #typeConstructor
-    | number                                                                              #numericLiteral
-    | booleanValue                                                                        #booleanLiteral
-    | string                                                                              #stringLiteral
-    | BINARY_LITERAL                                                                      #binaryLiteral
+    : literal                                                                             #literals
```

Comment:
This change is unnecessary (on its own). Squash it with the commit that introduces the DEFAULT clause.